src/components/BulkEnrollmentPage/index.test.jsx

```diff
@@ -1,94 +1,138 @@
 import React from 'react';
 import { act, screen } from '@testing-library/react';
 import configureMockStore from 'redux-mock-store';
-import { MemoryRouter } from 'react-router-dom';
 import thunk from 'redux-thunk';
 import { Provider } from 'react-redux';
+import userEvent from '@testing-library/user-event';
 import { renderWithRouter } from '../test/testUtils';
 import BulkEnrollmentPage from './index';
 import '../../../__mocks__/react-instantsearch-dom';
 import LicenseManagerApiService from '../../data/services/LicenseManagerAPIService';
 import { ROUTE_NAMES } from '../EnterpriseApp/constants';
 import { ADD_COURSES_TITLE } from './stepper/constants';
+import '@testing-library/jest-dom';
 
 jest.mock('../../data/services/LicenseManagerAPIService', () => ({
   __esModule: true,
   default: {
     fetchSubscriptions: jest.fn(),
+    fetchSubscriptionUsersOverview: jest.fn(),
   },
 }));
 
+const testSlug = 'horsesRus';
+
 const sub1 = {
   uuid: 'foo',
   title: 'horse subscription',
-  enterpriseSlug: 'horsesRus',
+  enterpriseSlug: testSlug,
   startDate: 1621347682,
   expirationDate: 1721547682,
   licenses: {
     allocated: 5,
     total: 20,
   },
+  enterpriseCatalogUuid: 'blarghl2',
 };
 
 const sub2 = {
   uuid: 'bearz',
   title: 'bear subscription',
-  enterpriseSlug: 'horsesRus',
+  enterpriseSlug: testSlug,
   startDate: 1621347682,
   expirationDate: 1721547682,
   licenses: {
     allocated: 15,
     total: 40,
   },
+  enterpriseCatalogUuid: 'blarghl',
 };
 
+const mockSubscriptionCount = Promise.resolve({
+  data: [
+    { status: 'unassigned', count: 0 },
+    { status: 'assigned', count: 0 },
+    { status: 'activated', count: 2 }],
+});
 const subscriptions = Promise.resolve({
   data: {
-    results: {
-      subscriptions: [sub1, sub2],
-    },
+    results: [sub1, sub2],
     errors: null,
     setErrors: jest.fn(),
     forceRefresh: jest.fn(),
     loading: false,
+    count: 2,
   },
 });
-
-LicenseManagerApiService.fetchSubscriptions.mockReturnValue(subscriptions);
+const singleSubscription = Promise.resolve({
+  data: {
+    results: [sub1],
+    errors: null,
+    setErrors: jest.fn(),
+    forceRefresh: jest.fn(),
+    loading: false,
+    count: 1,
+  },
+});
+const mockSingleSubscriptionCount = Promise.resolve({
+  data: [
+    { status: 'unassigned', count: 0 },
+    { status: 'assigned', count: 0 },
+    { status: 'activated', count: 1 }],
+});
 
 const fakeStore = {
   portalConfiguration: {
     enterpriseId: 'foo',
+    enableCodeManagementScreen: false,
+    enableSubscriptionManagementScreen: true,
```

Comment:
I think I added these to suppress some of the stupid prop warnings.
There are still a bunch of them because they get generated before the promises resolve. 😑

Reply:
If the components are attempting to render while promises are resolving, that sounds like a thing we should actually _fix_. Like, with a loading state.

Reply:
If that doesn't make sense as part of this ticket, let's open a ticket for it? We shouldn't have a bunch of props errors in our test and definitely not in our console when the page is loading.

Reply:
These particular lines are for props that would normally exist at the very beginning of rendering the EnterpriseApp and get carried all the way through, so it seems fine to add them here as they are indeed expected.

For the others I mention, we have various loading states in a lot of spots, but it's likely we just aren't doing anything with them between components. Like I know we have a loading state in the Subscription Context we're wrapping everything with here, however, we may not be using that loading state in the child components. It's probably worth a separate ticket.